oras/auth/token.py

```diff
@@ -152,7 +151,7 @@ def request_anonymous_token(self, h: auth_utils.authHeader) -> bool:
         if h.scope:
             params["scope"] = h.scope
 
-        logger.debug(f"Final params are {params}")
+        logger.debug(f"Requesting anon token with params: {params}")
```

Comment:
Just a sanity check - could this have any parameters we wouldn't want to be printed? I would probably leave this line out entirely.

Reply:
only `service` and `scope`:

https://github.com/tarilabs/oras-py/blob/19f452836513c572c248027352a5d95eaefd4337/oras/auth/token.py#L148-L152

which to my understanding are not sensitive info and done analogously for the auth token.

Just to be clear, I've just changed the hardcoded text to make it significant it's about an anon token when observing the logs, and the `params` was there before :)

I thought aligning log statements in:

- https://github.com/tarilabs/oras-py/blob/7d891e96e24f16f690be4c3953a53cbd819dfe6f/oras/auth/token.py#L127-L128
- https://github.com/tarilabs/oras-py/blob/7d891e96e24f16f690be4c3953a53cbd819dfe6f/oras/auth/token.py#L154-L155

to your comment of:

> ... put this debug message in the function that does that, and put a bit more metadata to make the comment more useful ...

([source](https://github.com/oras-project/oras-py/pull/148#discussion_r1771573650))

but let me know if you prefer me to change those log debug statement to something else!

Reply:
> only service and scope:

I just wanted to double check this - I think this should be ok to print.